pkg/cmd/server/api/types.go

```diff
@@ -116,6 +117,23 @@ type NodeConfig struct {
 
 	// IPTablesSyncPeriod is how often iptable rules are refreshed
 	IPTablesSyncPeriod string
+
+	// VolumeConfig contains options for configuring volumes on the node.
+	VolumeConfig VolumeConfig
+}
+
+// VolumeConfig contains options for configuring volumes on the node.
+type VolumeConfig struct {
+	// LocalQuota contains options for controlling local volume quota on the node.
+	LocalQuota LocalQuota
+}
+
+// LocalQuota contains options for controlling local volume quota on the node.
+type LocalQuota struct {
+	// FSGroup can be specified to enable a quota on local storage use per unique FSGroup ID.
+	// At present this is only implemented for emptyDir volumes, and if the underlying
+	// volumeDirectory is on an XFS filesystem.
+	FSGroup *resource.Quantity
```

Comment:
It was but this was requested way above by David, it lets us use things like 256Mi / 2Gi in the node-config.yaml setting, keeps everything more consistent with the other places we use Quantity.